### PR TITLE
[SCB-462] cloud eye publisher switch to new mechanism

### PR DESCRIPTION
**Pull request by wujimin**
image

### CODE REVIEW

File: samples/metrics-write-file-sample/metrics-write-file/src/main/resources/sample-log4j2.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<Configuration status="INFO">
```
Comment (WillemJiang) [inline samples/metrics-write-file-sample/metrics-write-file/src/main/resources/sample-log4j2.xml]: License header please.

Comment (wujimin) [reply samples/metrics-write-file-sample/metrics-write-file/src/main/resources/sample-log4j2.xml]: done

File: samples/metrics-write-file-sample/metrics-write-file/src/main/java/org/apache/servicecomb/samples/mwf/CloudEyeMetricModel.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  @JsonAnyGetter
+  public Map<String, String> getDynamicValue() {
+    return dynamicValue;
+  }
+
+  public void setDynamicValue(Map<String, String> dynamicValue) {
+    this.dynamicValue = dynamicValue;
+  }
+
+  public static void main(String[] args) throws IOException {
```
Comment (liubao68) [inline samples/metrics-write-file-sample/metrics-write-file/src/main/java/org/apache/servicecomb/samples/mwf/CloudEyeMetricModel.java]: I think this test case can be removed

Comment (wujimin) [reply samples/metrics-write-file-sample/metrics-write-file/src/main/java/org/apache/servicecomb/samples/mwf/CloudEyeMetricModel.java]: stupid mistake
fixed

**Pull request merged**